packages/core/src/sass/account-wizard.scss

```diff
@@ -485,6 +485,11 @@
     display: flex;
     flex-direction: column;
 
+    .dc-radio-group {
+        flex-direction: column;
+        align-items: flex-start;
+    }
+
```

Comment:
Why do you need this in 2 different place?

Reply:
this one is to make it vertically for trading assessment because by default the button groups is horizontal.